docs/authzn.adoc

```diff
@@ -276,3 +276,13 @@ sec-roles: ROLE_ORG_6007280321;ROLE_GDI_PLANER;ROLE_GDI_EDITOR;ROLE_USER
 sec-org: 6007280321
 ```
 
+== External authentication
+Whenever an external authentication is used (OAuth2 or external IDP), a new attribute is added to Header, named :
+```
+sec-external-authentication
+```
+which is set to "true" in this case.
+
+As a result, information such as login ID will not be shown on the application.
+
+Also there is no possibility to change user password in such case since user is managed by an external entity.
```

Comment:
it's not under the gateway's scope what/how the info will be consumed by proxified webapps (e.g. console or whatever else).  So I'd have put it a bit differently:

```
This allows the proxified webapps to adapt their behaviour consequently: as an example, it does not make sense to display a password update form in the geOrchestra console if the user is logged in via a third party identity provider. Having the flag passed in the HTTP headers allows to enable or disable such a form.
```